plone/app/content/tests/test_widgets.py

```diff
@@ -680,6 +680,24 @@ def testGetMimeIcon(self):
             [{"getMimeIcon": "/plone/++resource++mimetype.icons/unknown.png"}],
         )
 
+    def testScrubHtml(self):
+        from zope.schema.vocabulary import SimpleTerm
+        from zope.schema.vocabulary import SimpleVocabulary
+
+        view = VocabularyView(self.portal, self.request)
+        vocab = SimpleVocabulary(
+            [
+                SimpleTerm(
+                    token=f"term {idx} <b>",
+                    value=f"term {idx} <b>",
+                    title=f"term {idx} <b>",
+                )
+                for idx in range(3)
+            ]
+        )
+        with mock.patch.object(view, "get_vocabulary", return_value=vocab):
+            json.loads(view())
```

Comment:
This would be a better test if it made an assertion on the result to confirm that the html was scrubbed as expected.

Reply:
There are other tests that do that. The point of this test is to check that the json can be parsed. A test like this was missing before. I would keep it simple and not do more checks in this test.

Reply:
Maybe it should be called `testJson` to make this clearer?